core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/dot-uve-toolbar/dot-uve-toolbar.component.ts

```diff
@@ -90,7 +89,9 @@ export class DotUveToolbarComponent implements OnInit {
     readonly $apiURL = this.#store.$apiURL;
     readonly $personaSelectorProps = this.#store.$personaSelector;
     readonly $infoDisplayProps = this.#store.$infoDisplayProps;
-    readonly $devices: WritableSignal<DotDeviceListItem[]> = signal([]);
+    readonly $devices: Signal<DotDeviceListItem[]> = toSignal(
+        this.#deviceService.get().pipe(map((devices = []) => [...DEFAULT_DEVICES, ...devices]))
```

Comment:
Using the toSignal function should include an initialValue or mark the signal as requireSync

Reply:
Taking notes!